README.md

```diff
@@ -118,7 +118,9 @@ export const store = useRootModule({
 })
 ```
 
-Explicit typing:
+#### Undefined handling
```

Comment:
@abemscac Does this description sound correct to you?

Reply:
Yes, I think so.
The current behavior of a module is (assuming we have a state `{ id: number, name: string }` in a module):

1. Module is initialized with `{ id: 1, name: 'hello' }`, which makes `id` and `name` to exist in the flattened module.
2. User updates the value of `name` to `undefined`.
  - Meanwhile, LocalStoragePlugin will write the new state into LocalStorage using `JSON.stringify`, so `name` will not appear in the state (string) in LocalStorage, only `"{"id":1}"`.
4. `id` and `name` still exist in the flattened module (in memory), and the value is `{ id: 1, name: undefined }`.
5. User reloads (refreshes) the whole page, which makes LocalStoragePlugin to load the stringified states from LocalStorage and convert them to objects using `JSON.parse`.
6. Since the stringified state is `"{"id":1}"`, `name` will not be in the flattened module (memory) anymore. Any usage of `name.value` will lead to `Cannot read properties of undefined (reading 'value')` error.